server/src/helpers/translation.js

```diff
@@ -37,7 +39,9 @@ const getTranslatedText = async (text, originalLanguage) => {
     return { language, text: data.translations[0].translatedText }
   } catch (error) {
     // eslint-disable-next-line no-console
-    console.error(error)
+    console.error(
+      "L'appel à Google Cloud  Translation API a échoué. Vérifiez les limites d'appels fixées pour ce projet"
```

Comment:
C'est important de logguer quand même l'erreur sous-jacente
```suggestion
      "L'appel à Google Cloud Translation API a échoué. Vérifiez les limites d'appels fixées pour ce projet.", error
```